### DISCUSSION TITLE
Increasing max Encumbrance and Corruption Points

### DISCUSSION THREAD

**Discussion by ADDNW** [moo-man/WFRP4e-FoundryVTT#1535] [Q&A]
At start - I don't own the modules with talents so I made talents myself.

But coming to talents increasing those max values - how can I make them work?
Is it an Passive Effect I have to implement, or is there somewhere a simple option to make it?

**Comment by moo-man** [comment]
I am out of town for a few days. Your best resource would be the Discord channels, as you can likely talk with others about how to create effects
